[E2E] [bug] VM abort migration lack check points

## What's the test to develop? Please describe
The test case `test_abort_migrate` failed in following builds:
- ECM Lab, `harvester-install-and-test build#33`
- ECM Lab, `harvester-install-and-test-e2e-daily`, test on **master** branch

## Describe the items of the test development (DoD, definition of done) you'd like
### Error messages
1. In ECM Lab `daily#111`: 500, "can't migrate the VM to the node harvester-jqmk2: no matching migratable nodes found"
2. In ECM Lab `daily#113`: 500, "Can't migrate the VM, the VM is not in ready status"
3. In ECM Lab `install-and-test#33`
    ```
    AssertionError: Failed to abort VM(vm-13h18m32s635693-03-05)'s migration, stuck on Aborting migration
    API Status(200)
    ```

we mostly hit 3rd error message, but the function is working when test it manually.

--- a/harvester_e2e_tests/integrations/test_3_vm_functions.py
+++ b/harvester_e2e_tests/integrations/test_3_vm_functions.py
@@ -512,6 +512,9 @@ class TestVMOperations:
                     assert 204 == code, (code, err)
                 else:
                     break
+            elif len(states) == 1 and not m_state:
+                # we did abort migration, and the anootation removed
+                break
             sleep(3)
         else:
             raise AssertionError(

--- a/harvester_e2e_tests/integrations/test_z_terraform.py
+++ b/harvester_e2e_tests/integrations/test_z_terraform.py
@@ -45,7 +45,7 @@ def vlanconfig_resource(request, unique_name, tf_resource, clusternetwork_resour
     assert vlan_nic, f"VLAN NIC {vlan_nic} not configured correctly."
 
     _, clusternetwork_name = clusternetwork_resource
-    name, nics = f"{clusternetwork_name}-{vlan_nic}", [vlan_nic]
+    name, nics = f"{clusternetwork_name}-{vlan_nic}".lower(), [vlan_nic]
     spec = tf_resource.vlanconfig(f"tf_{unique_name}", name, clusternetwork_name, nics)
 
     return spec, name, clusternetwork_name, nics

